packages/clarity-js/types/data.d.ts

```diff
@@ -146,6 +146,20 @@ export interface BaselineData {
     screenHeight: number;
 }
 
+export interface TimelineState {
```

Comment:
These changes may not be necessary if we don't need to include Mutation as part of the timeline.